06-reactjs/components/movie-row.js

```diff
@@ -0,0 +1,21 @@
+var React = require('react');
```

Comment:
Yes, you're right anyway maybe he can make a table component and a table-row component a more generic component lol :P